src/masternodes/rpc_vault.cpp

```diff
@@ -0,0 +1,474 @@
+#include <masternodes/mn_rpc.h>
+
+extern UniValue AmountsToJSON(TAmounts const & diffs);
+
+namespace {
+    UniValue VaultToJSON(const CVaultMessage& vault, const CVaultId& id) {
+        UniValue collateralBalances{UniValue::VARR};
+        UniValue loanBalances{UniValue::VARR};
+
+        if(auto collateral = pcustomcsview->GetVaultCollaterals(id))
+            collateralBalances = AmountsToJSON(collateral->balances);
+
+        if(auto loan = pcustomcsview->GetLoanTokens(id))
+            loanBalances = AmountsToJSON(loan->balances);
+
+        UniValue result{UniValue::VOBJ};
+        result.pushKV("loanschemeid", vault.schemeId);
```

Comment:
I would keep the same format for all keys we use. first word have small letter, each other word have first letter capital:
loanSchemeId, ownerAddress ...

Reply:
True there is a lot of inconsistencies in rpc inputs and outputs... I'll change this but we need further discussion